Makefile-az.mk

```diff
@@ -38,6 +38,18 @@ az-mkacr: az-mkrg ## Create test ACR
 		--sku Basic --admin-enabled -o none
 	az acr login  --name $(AZURE_ACR_NAME)
 
+az-acrimport: ## Imports an image to an acr registry
+	az acr import --name $(AZURE_ACR_NAME) --source "mcr.microsoft.com/oss/kubernetes/pause:3.6" --image "pause:3.6"
+
+az-cleanenv: 
+	kubectl delete nodepools --all
+	for nodeclaim in $$(kubectl get nodeclaims --output=jsonpath={.items..metadata.name}); do \
+		kubectl patch nodeclaim $$nodeclaim --type=json -p '[{"op": "remove", "path": "/metadata/finalizers"}]'; \
```

Comment:
I think if the nodepools still exist (deleted after), and all the pods are gone where we don't have to drain and such, probably don't need to remove the finalizers? 

However, deleting them seems fine too?